Packs/AbnormalSecurity/ReleaseNotes/2_3_2.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Abnormal Security Event Collector
+
+- Implemented event fetching limit to prevent exceeding the maximum fetch events threshold in the API.
```

Comment:
Fixed an issue where **fetch-events** failed due to exceeding the fetching limit threshold.